src/components/Header/Header.jsx

```diff
@@ -1,6 +1,12 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Navigation from '../Navigation/Navigation';
```

Comment:
better create index.js in all directories to shorten your imports
```suggestion
import Navigation from '../Navigation';
```